static/style.css

```
@@ -887,6 +887,71 @@ body {
+    background: #11161f;
+    font-family: 'Courier New', monospace;
+    font-size: 0.82rem;
+    line-height: 1.35;
+}
+
+.backend-log-line {
+    padding: 6px 12px;
+    border-bottom: 1px solid rgba(255, 255, 255, 0.06);
+    color: #d6dde8;
+    white-space: pre-wrap;
+    word-break: break-word;
```

Stylelint (17.4.0)

[error] Unexpected deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)